### PR TITLE
Customize UX and copy

### PR DESCRIPTION
**Pull request by GuillaumeDelente**
- Update maven gradle plugin to allow project to build.
- Disable photos/videos dropdown.
- For multi-photo selection, have a tap on the image select the photo as opposed to open the preview.
- Update copy.

### CODE REVIEW

File: app/src/main/java/com/tangxiaolv/telegramgallery/app/MainActivity.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
             @Override
             public void onClick(View v) {
                 GalleryConfig config = new GalleryConfig.Build()
-                        .limitPickPhoto(3)
+                        .limitPickPhoto(11)
```
Comment (kamranpirwani) [inline app/src/main/java/com/tangxiaolv/telegramgallery/app/MainActivity.java]: Is this what we agreed on?

File: telegramgallery/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
     <string name="ClearButton">Clear</string>
 
+    <string name="SearchImagesTitle">Search web</string>
+    <string name="SearchGifsTitle">Search GIFs</string>
+
     <string name="SearchImagesInfo">WEB SEARCH</string>
-    <string name="MostSelect">你最多只能选择%d张图片</string>
+    <string name="MostSelect">You can select up to %d pictures</string>
 
-    <string name="Send">完成</string>
```
Comment (kamranpirwani) [inline telegramgallery/src/main/res/values/strings.xml:27]: Hahah

**Review by kamranpirwani — APPROVED** [review]
Approved. We should sync up with product to figure out the hard limit on number of uploads

**Comment by GuillaumeDelente** [comment]
@kamranpirwani yep. Change made here regarding the max image selected count are only for the sample app and will not have any impact on the library itself.

**Pull request merged**